From 98e34a4215f24a2414ca34a65804c86b8e4405c8 Mon Sep 17 00:00:00 2001
From: Peter Eisenmann <p3732@getgoogleoff.me>
Date: Wed, 30 Nov 2022 02:40:17 +0100
Subject: [PATCH] help-overlay: drop tree view shortcuts

Expanding folders in list view is not implemented in version 43. This
removes the shortcuts from the keyboard shortcut help view.

Fixes #2674
---
 src/resources/gtk/help-overlay.ui | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/src/resources/gtk/help-overlay.ui b/src/resources/gtk/help-overlay.ui
index 27e5906b6e..42bad9ffd1 100644
--- a/src/resources/gtk/help-overlay.ui
+++ b/src/resources/gtk/help-overlay.ui
@@ -301,20 +301,6 @@
                 <property name="accelerator">&lt;Primary&gt;2</property>
               </object>
             </child>
-            <child>
-              <object class="GtkShortcutsShortcut">
-                <property name="visible">True</property>
-                <property name="title" translatable="yes" context="shortcut window">Expand folder</property>
-                <property name="accelerator">Right</property>
-              </object>
-            </child>
-            <child>
-              <object class="GtkShortcutsShortcut">
-                <property name="visible">True</property>
-                <property name="title" translatable="yes" context="shortcut window">Collapse folder</property>
-                <property name="accelerator">Left</property>
-              </object>
-            </child>
           </object>
         </child>
         <child>
-- 
GitLab

